conf/metrics.properties.template

```diff
@@ -47,6 +47,10 @@
 #    instance master and applications. MetricsServlet may not be configured by self.
 #
 
+## List of available common sources and their properties.
+
+# org.apache.spark.metrics.source.JvmSource
```

Comment:
It's the only Common Source users can add additionally, all the spark internal source are added automatically. 
No value for this one, just as `org.apache.spark.metrics.sink.JmxSink`.